core/allocators.h

```diff
@@ -37,20 +37,17 @@ template <int PREALLOC_COUNT = 64, int MAX_HANDS = 8>
 class BalloonAllocator {
 
 	enum {
-
 		USED_FLAG = (1 << 30),
 		USED_MASK = USED_FLAG - 1
 	};
 
 	struct Balloon {
-
-		Balloon *next;
 		Balloon *prev;
+		Balloon *next;
```

Comment:
Is there a reason to change this?

Reply:
There's no need to change something like that. It creates noise in git blame, for no gain

Reply:
I have no experience with `git blame`, so can't comment on the noisiness it might add.
But the way I see it, is a choice between:
- a noiser `git blame`
or
- logical ordering.

One small change may seem insignificant, but many small changes of a similar nature can make the code base more pleasant to work with.

Reply:
git blame looks like this: https://github.com/godotengine/godot/blame/e0033df4db8b399e40682866f56bf8249ac96144/core/allocators.h
It can be useful for getting a quick idea of how a file has changed and sometimes a good starting point when you're hunting for a recent regression.
Regardless of your decision here, it's worth having a play with blame to see if you might find it useful.

Reply:
I really should learn more about git blame, as it has been mentioned before; when I've made small changes.
Thanks for the link :)